Vizzuality/heco-invest
# feat: Added impact_calculated attribute to Project model

I am adding new attribute called `impact_calculated` to Project model and its serializer which can be used by FE (@barbara-chaves ) to decide if impact already was calculated for appropriate Project.

## Testing instructions

Via Rswag -- check that Project contains correct value at `impact_calculated` attribute. The easiest approach is probably to change geometry or impact areas of already existing project --> immediately after change, this attribute is gonna be false, but changes to true after impacts are recalculated.

## Tracking

https://vizzuality.atlassian.net/browse/LET-872
